web_responsive/static/src/js/web_responsive.js

```diff
@@ -310,60 +310,57 @@ odoo.define('web_responsive', function(require) {
         },
     });
 
-    // FieldStatus toggle hidden-xs
+    // FieldStatus (responsive fold)
     RelationalFields.FieldStatus.include({
-        _render: function () {
+        _setState: function () {
             this._super.apply(this, arguments);
-            this.$el.find('button.btn-primary.disabled').css('pointer-events', 'initial');
+            _.each(this.status_information, function(item){
+                item.fold = !item.selected;
+            });
         },
 
-        _onClickStage: function (e) {
-            this._super(e);
-            this.$el.find('button.btn-default').toggleClass('hidden-xs');
-            $(e.currentTarget).blur();
-        },
+        _render: function () {
+            // Break inheritance!
```

Comment:
Is there a chance to break it in the qweb template instead, by replacing elements?